TicketManagementSystem/src/scripts/handlers/handleTicketPurchase.js

```diff
@@ -0,0 +1,12 @@
+import { fetchSaveOrder } from "../network/saveOrder";
+export async function handleTicketPurchase(ticketCategoryId, numberOfTickets, eventId, api_urls){
+    // manually entered userId
+    let userId = '8A029008-1AAF-46E3-B319-0006D111F229';
```

Comment:
If you don't change the value, you can use const.
```suggestion
    const userId = '8A029008-1AAF-46E3-B319-0006D111F229';
```